modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/components/SlaCard/index.js

```diff
@@ -0,0 +1,162 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * The contents of this file are subject to the terms of the Liferay Enterprise
+ * Subscription License ("License"). You may not use this file except in
+ * compliance with the License. You can obtain a copy of the License by
+ * contacting Liferay, Inc. See the License for the specific language governing
+ * permissions and limitations under the License, including but not limited to
+ * distribution rights of the Software.
+ */
+
+import ClayIcon from '@clayui/icon';
+import classNames from 'classnames';
+import {useEffect, useState} from 'react';
+import {FORMAT_DATE} from '../../../../common/utils/constants/slaCardDate';
+import {SLA_NAMES} from '../../../../common/utils/constants/slaCardNames';
+import SlaCardLayout from './Layout';
+
+const SlaCard = ({project}) => {
+	const [slaData, setSlaData] = useState();
+	const [slaSelected, setSlaSelected] = useState('');
+	const SLA_DATA = {};
+
+	useEffect(() => {
+		if (project) {
+			const slaFiltedData = [];
+			const slaNewDate = (slaDate) => {
+				return new Date(slaDate)?.toLocaleDateString(
+					'en-US',
+					FORMAT_DATE
+				);
+			};
+
+			const slaRawData = {
+				current: {
+					slaDateEnd: slaNewDate(project.slaCurrentEndDate),
+					slaDateStart: slaNewDate(project.slaCurrentStartDate),
+					slaLabel: SLA_NAMES.current.toUpperCase(),
+					slaTitle: project.slaCurrent?.split(' ')[0],
+				},
+				expired: {
+					slaDateEnd: slaNewDate(project.slaExpiredEndDate),
+					slaDateStart: slaNewDate(project.slaExpiredStartDate),
+					slaLabel: SLA_NAMES.expired.toUpperCase(),
+					slaTitle: project.slaExpired?.split(' ')[0],
+				},
+				future: {
+					slaDateEnd: slaNewDate(project.slaFutureEndDate),
+					slaDateStart: slaNewDate(project.slaFutureStartDate),
+					slaLabel: SLA_NAMES.future.toUpperCase(),
+					slaTitle: project.slaFuture?.split(' ')[0],
+				},
+			};
```

Comment:
I guess that we already have a function that return a date formatted as this one. Check this.
If not, should be good change it name.

to simplify, you can also destructure the project object in other variables.

eg.
```
const {slaCurrentEndDate, slaCurrentStartDate} = project;
```

```suggestion
			const getFormatDate = (slaDate) => {
				return new Date(slaDate).toLocaleDateString(
					'en-US',
					FORMAT_DATE
				);
			};

			const slaRawData = {
				current: {
					slaDateEnd: project.slaCurrentEndDate && getFormatDate(project.slaCurrentEndDate),
					slaDateStart: project.slaCurrentStartDate && getFormatDate(project.slaCurrentStartDate),
					slaLabel: SLA_NAMES.current.toUpperCase(),
					slaTitle: project.slaCurrent?.split(' ')[0],
				},
				expired: {
					slaDateEnd: project.slaExpiredEndDate && getFormatDate(project.slaExpiredEndDate),
					slaDateStart: project.slaExpiredStartDate && getFormatDate(project.slaExpiredStartDate),
					slaLabel: SLA_NAMES.expired.toUpperCase(),
					slaTitle: project.slaExpired?.split(' ')[0],
				},
				future: {
					slaDateEnd: project.slaFutureEndDate && getFormatDate(project.slaFutureEndDate),
					slaDateStart: project.slaFutureStartDate && getFormatDate(project.slaFutureStartDate),
					slaLabel: SLA_NAMES.future.toUpperCase(),
					slaTitle: project.slaFuture?.split(' ')[0],
				},
			};
```

Reply:
I will change the name, because the function what exist bring different date format. Thank you.

Reply:
About `const {slaCurrentEndDate, slaCurrentStartDate} = project;` I I destroyed the project object with `slaCurrentEndDate, slaCurrentStartDate, slaExpiredEndDate, slaExpiredStartDate, slaFutureEndDate, slaFutureStartDate`